malariagen_data/anoph/to_plink.py

```diff
@@ -0,0 +1,164 @@
+from typing import Optional
+
+import allel  # type: ignore
+import numpy as np
+import os
+import bed_reader
+
+from util import dask_compress_dataset
```

Comment:
```suggestion
from ..util import dask_compress_dataset
```

Fix relative import.